### ISSUE TITLE
card background color

### ISSUE THREAD

**Issue by tomg1970** [custom-cards/bignumber-card#34]
Hello, can I also change the card background. 
So the background colour of the card itself is not the `bnStyle`.

**Comment by Jeppedy** [comment]
I agree, it'd be nice.  But I think this project is no longer maintained...
